js-exercises/is-triangle/isTriangle.js

```diff
@@ -0,0 +1,53 @@
+function isNumber(...args) {
+  let result = true;
+
+  for (const arg of args) {
+    if (typeof arg !== 'number') {
+      result = false;
+      break;
+    }
+  }
+
+  return result;
+}
+
+function isPositiveInteger(...args) {
+  let result = true;
+
+  for (const arg of args) {
```

Comment:
Would this work?
```suggestion
args.every(arg => arg > 0)
```

Reply:
Yes, changed in latest commit with `every`